docs/index.md

```diff
@@ -3,41 +3,266 @@ hide:
+      </div>
+    </a>
+  </li>
+</ul>
 
-The **Graphical User Interface** of Taipy allows anyone with basic knowledge of Python to create a beautiful and
-interactive interface. It is a simple and intuitive way to create a GUI. No need to know how to design web pages
-with CSS or HTML. Taipy uses an augmented syntax of Markdown to create your desired Web page.
+## How does it work?
 
-The **Scenario Management** component of Taipy is a powerful tool to manage business problems and pipelines. Using
-**Taipy Core** is straightforward. Among other features, you are able to:
+Taipy is composed of two main independent components: **Taipy Core** and **Taipy GUI**. You can use the GUI component without the Scenario Management and vice-versa. However, as you will see, they are incredibly efficient when combined.
 
-- Keep track of your KPI, data, scenarios, pipelines, etc.
-- Have smart scheduling
-- Simplify industrialization for Data Visualization, Machine-Learning, Optimization, etc.
+<div class="tp-row" style="margin-top: 2rem; margin-bottom: 2rem">
+  <div class="tp-col-12 tp-col-md">
+    <article class="tp-content-card">
+      <h3>Taipy GUI</h3>
+      <p>
+        The <strong>Graphical User Interface</strong> of Taipy allows anyone with basic knowledge of Python to create a beautiful and
+        interactive interface. It is a simple and intuitive way to create a GUI. No need to know how to design web pages
+        with CSS or HTML. Taipy uses an augmented syntax of Markdown to create your desired Web page.
+      </p>
+    </article>
+    <article class="tp-content-card">
+      <h3>Taipy Core</h3>      
+      <p>The <strong>Scenario Management</strong> component of Taipy is a powerful tool to manage business problems and pipelines. Using <strong>Taipy Core</strong> is straightforward. Among other features, you are able to:</p>
+      <ul>
+        <li>Keep track of your KPI, data, scenarios, pipelines, etc.</li>
+        <li>Have smart scheduling</li>
+        <li>Simplify industrialization for Data Visualization, Machine-Learning, Optimization, etc.</li>
+      </ul>
+      <p>You can use the GUI component without the Scenario Management and vice-versa. However, as you will see, they are incredibly efficient when combined.</p>
+    </article>
+  </div>
 
-You can use the GUI component without the Scenario Management and vice-versa. However, as you will see, they are
-incredibly efficient when combined.
+  <div class="tp-col-12 tp-col-md-auto">
+    <figure align="center">
+      <img alt="Taipy structure" src="images/taipy-structure-vector.svg" width="350">
+    </figure>
+  </div>
 
-Other packages offer additional functionality, such as a **REST API** that allows for deploying
-Taipy application on Web architectures.
+</div>
 
-The documentation for Taipy is divided in three main sections:
+Other packages offer additional functionality, such as a **REST API** that allows for deploying Taipy application on Web architectures.
```

Comment:
"for deploying Taipy application on Web architectures"
->
"for deploying Taipy Core applications as a Web service"

Reply:
Done